crates/ty_python_semantic/resources/mdtest/binary/tuples.md

```diff
@@ -17,6 +17,9 @@ def _(x: tuple[int, str], y: tuple[None, tuple[int]]):
 
 ```py
 def _(x: tuple[int, ...], y: tuple[str, ...]):
-    reveal_type(x + y)  # revealed: @Todo(full tuple[...] support)
-    reveal_type(x + (1, 2))  # revealed: @Todo(full tuple[...] support)
+    # error: [unsupported-operator] "Operator `+` is unsupported between objects of type `tuple[int, ...]` and `tuple[str, ...]`"
+    reveal_type(x + y)  # revealed: Unknown
+
+    # TODO: should be `tuple[int, ...]`
+    reveal_type(x + (1, 2))  # revealed: tuple[_T_co, ...]
```

Comment:
this looks similar to https://github.com/astral-sh/ruff/pull/17832#discussion_r2081502056